vms/vms/settings.py

```diff
@@ -14,11 +14,11 @@
 SECRET_KEY = 'rs473)3n^fe0^t-^s$n)_%pl=75f_na7z5ee@(^xc-vn^bzr%a'
 
 # SECURITY WARNING: run with debug turned off (DEBUG = False) in production!
-DEBUG = True
+DEBUG = False
```

Comment:
We shouldn't be pushing this, you can remove all changes done to settings.py for this issue.